### ISSUE TITLE
[Feature] Let redstone control chunk loaders

### ISSUE THREAD

**Issue by SabrCat** [SuperMartijn642/ChunkLoaders#110]
**Is your feature request related to a problem? Please describe.**
The issue we run into on my create server is that a lot of factories need to be chunk loaded for the system to work. However, most factories only need to actually be loaded if they are working, which is less than 5% of the time. The rest of the time they needlessly consume precious server resources without being used, just so they are available if an automated system calls them.

**Describe the solution you'd like**
If we had Chunk loaders that could be switched on or off with redstone, we could have one permanently loaded chunk that monitors the Output storage of the factory, and only when that storage goes low enable the chunk loading for the entire factory, which means we'd only use up the server resources when they are actually needed. This would make our plans for a serverwide networked automated system possible, as it currently runs into lag limits far before we even have half of it working.

**Comment by Flavio-coding** [comment]
Yes, I'd really love this feature. I love making storage systems and farm stuff too, but keeping the water transport system always loaded is a bit of a pain for my laptop. 
I really love this mod, but it would be really cool being able to something like a control room item that can enable and disable in bulk some chunks, or if it's too complicated a command would be more than enough for a normal player.
